testing/trino-benchto-benchmarks/src/test/java/io/trino/sql/planner/AbstractCostBasedPlanTest.java

```diff
@@ -84,6 +85,8 @@ protected LocalQueryRunner createLocalQueryRunner()
 
     protected abstract InMemoryMetastoreFactory getMetastoreFactory();
 
+    protected abstract Map<String, String> getPartitionColumns();
```

Comment:
I think `getQueryPlanResourcePath` should be an abstract method and the derived class should provide the right path instead of making the base class aware about partitioning